cspell.json

```diff
@@ -1,7 +1,16 @@
 {
```

Comment:
Renaming doesn't seem right

Reply:
At some point cspell wouldn't want to pickup its configuration, and removing the dot fixed it. It seems it came back to its senses, I reverted.